src/askvm/index.ts

```diff
@@ -1,4 +1,34 @@
-export * from './lib';
-export { resources, types };
+import type { AskCodeOrValue } from '../askcode';
+import { extendOptions, run as libRun, untyped } from './lib';
+import type { JSONable, Options, TypedValue } from './lib';
 import * as types from './lib/type';
 import * as resources from './resources';
+export { extendOptions, resource, typed, untyped } from './lib';
+export type { Options, Resource, Resources, Values } from './lib';
+export { resources, types };
+
+export async function run(
+  inputOptions: Options,
+  code: AskCodeOrValue,
+  args?: any[]
+): Promise<TypedValue<JSONable>> {
+  const options = extendOptions(inputOptions, {
+    stats: {
+      steps: 0,
+    },
+    limits: {
+      steps: 10 ** 4,
+    },
+  });
+  const result = await libRun(options, code, args);
+  // console.log('stats after run', options.stats);
```

Comment:
leave it here to help with future debugging - this will eventually be replaced with a logging solution

Reply:
ok, fine